wiki/Legal/SCTL/en.md

```diff
@@ -0,0 +1,37 @@
+---
+legal: true
+---
+
+# The following information is disclosed based on Article 11 of the "Act on Specified Commercial Transactions"
+
+## Name of the seller
+
+ppy Pty Ltd ACN 163 593 413 a.t.f. Dean Herbert Family Trust
+
+## Location of the seller
+
+41 Gregory Street, Wembley, WA, 6014, Australia
+
+## Manager of the seller
+
+Director: Dean Herbert
+
+## Sales price
+
+Please check the purchase page on the site.
+
+## Payment methods
+
+You can use any of the following payment methods:
+
+Credit card, cash payment, Paypal, Docomo, NetCash, BitCash
+
+## Cancellation
+
+We accept cancellation and refund requests per customer's discretion. Please contact [accounts@ppy.sh](mailto:accounts@ppy.sh).
+
+## Contact
+
+Help and support: https://osu.ppy.sh/wiki/ja/Main_page
```

Comment:
not clickable otherwise

![image](https://github.com/ppy/osu-wiki/assets/36758269/5a980a20-2998-45bd-959b-f595aa25ca3f)

```suggestion
Help and support: <https://osu.ppy.sh/wiki/ja/Main_page>
```

Reply:
is the english page also required to link to the japanese main page? 🤔🤔🤔

Reply:
again, this is only for japanese audiences. i only made the english page out of necessity due to how the wiki works.